packages/interface/src/__tests__/GraphQLLanguageService-test.js

```diff
@@ -33,7 +33,7 @@ describe('GraphQLLanguageService', () => {
 
   it('runs diagnostic service as expected', async () => {
     const diagnostics = await languageService.getDiagnostics(
-      'qeury',
+      'query',
```

Comment:
This isn't a typo - as you see in line 39, this should expect a diagnostics error. Did this test pass with this change?

Reply:
Interesting, yes the tests passed for this change, it because with `query` it yields the following syntax error:

```
[ { severity: 1,
    message: 'Syntax Error: Expected {, found <EOF>',
    source: 'GraphQL: Syntax',
    range: Range { containsPosition: [Function], start: [Object], end: [Object] } } ]
```

and with `qeury` it yields the following error:

```
[ { severity: 1,
    message: 'Syntax Error: Unexpected Name "qeury"',
    source: 'GraphQL: Syntax',
    range: Range { containsPosition: [Function], start: [Object], end: [Object] } } ]
```

So, the errors are different but since we are only checking for the error count, the test passes. 

Reverted it to `qeury` and added a new assertion to make it clearer that it is not a typo and make the test more robust.